src/Task/Git/CommitMessage.php

```diff
@@ -93,6 +93,18 @@ public function canRunInContext(ContextInterface $context): bool
         return $context instanceof GitCommitMsgContext;
     }
 
+    private static function withCommitMessage(string $errorMessage, string $commitMessage): string
+    {
+        return sprintf(
+            "%s%sOriginal commit message:%s%s",
```

Comment:
this one has no space after `:`.
This makes the tests fail.

See https://github.com/phpro/grumphp/pull/1108/files#diff-93ecaacb0c8dbc50a120fd5d5f863454fe131909bfe4ed88e4ea085772a9c03dL170

```diff
+ '%s%sOriginal commit message: %s%s',
- "%s%sOriginal commit message:%s%s"
```

Reply:
Wow, well observed